Proposed update to the Face toolbox

woodbe opened the pull request:
This is based on the planned changes to have the PAD-L2 face toolbox. This removed the full 3D while leaving the wearable mask (that is just taking a printed photo and wearing it like a mask)

This also tries to use the same structure from the fingerprint toolbox update in terms of naming/numbering.

woodbe committed:
Proposed update to the Face toolbox

This is based on the planned changes to have the PAD-L2 face toolbox. This removed the full 3D while leaving the wearable mask (that is just taking a printed photo and wearing it like a mask)

This also tries to use the same structure from the fingerprint toolbox update in terms of naming/numbering.

woodbe commented:
So a question on the mobile device camera. I was peeking at the website and you now have to go pretty far down the list to get to a camera below 100. I'm wondering if we should look at using a higher number at this point, like say 120 or 125. The highest are listing out in the high 150s (with one above 160), but there are a LOT of devices above 125. Just filtering to only show devices since 2021 gives 83 that would qualify at 125 and 94 at 120 (bumping up about 10 each if we look at 2020 instead of 2021).

While I don't see an issue with the 100 score, this is more me wondering that we set in 2022, and at that time there were about 90 devices that would meet that number. We are a few years later and it would seem that as part of our update, we should look at updating the requirements for the smartphone camera.

I'm not suggesting any other changes to the camera (or printers, etc), just the mobile device one.

woodbe committed:
Updates

Realized that I had moved sections that shouldn't have been moved, so this update puts them back.

gfiumara commented:
I wouldn't have any issue increasing the 100 Dxomark score. It does seem like nearly every new "non-budget" device from the past 3 years has been at least 100. Maybe ≥ !120-140?

woodbe commented:
Where DSLR is mentioned, add "mirrorless"

woodbe commented:
See if we can match roughly to the level of device to the score of 100 in 2022, to make a similar cutoff today.

woodbe commented:
So rough markers for the phone camera score above 100 based on phone generations:

Apple - 4 generations (iPhone 14 back to iPhone 11)
Samsung - 2 generations (S22 and S21)
Google - 3 generations (7 to 5, the 4a is listed, but it is at 100 and it came out just a few months before the 5, so I'm leaving it as 3 generations)

If we went with a score of 135, we would get these:

Apple - 4 gens (16 to 13, though not all models of the 13/14 would qualify, we would need to go to 130 to get all of them)
Samsung - 3 gens (but only the Ultra versions, at 130 would get all the S24 & S23 except the FEs, but that would largely match)
Google - 3 gens (9-7, but the 7a and Fold would not be covered, but would at 130)

I am going to put in 130 as this I think makes it slightly easier and still puts a pretty good bar. There would be 36 devices just from these 3 vendors that make 130 and 71 in total. There were 87 at the older number, but I think 71 is still a large enough number. If we really wanted to have the same number (roughly 90 available devices), we could go to 125, but I think 130 is a good spot at this point.

woodbe commented:
> Where DSLR is mentioned, add "mirrorless"

Interestingly enough, I actually didn't find any references to DSLR. I think (though I could be misremembering), we had this discussion when we published that the specific types of cameras were changing, and so we shouldn't specifically reference that so we wouldn't get caught up in that, and so I think we just removed the DSLR completely.

It's not in the overview or the inventory, so I think we were careful enough before to not need to do this now.

huylabs commented:
What is the rationale to exclude specific models by this criterion? An "old" model (e.g., iPhone 11 ) can pass all anti-spoofing tests but it would be excluded bytes criterion.

woodbe commented:
To be clear, this is for taking the pictures that would be used to generate the spoof, not in testing against it. So here what we would be saying is that you couldn't use the iPhone 11 to take the picture of the person you are trying to create the spoof of. Basically we are just setting a bar on the main camera not being "good enough" today (while it would have been fine 3 years ago). 

You could still test the iPhone 11 for its ability to resist the spoof.

huylabs commented:
Got it. Thanks for the clarification!

woodbe committed:
Updates to incorporate changes from call

This update includes the following:

- adds a new environmental condition section to the inventory, defined following the description from the L2 update
- adds the new environmental conditions to the requirements in the attacks
- adjusts the overview to reference the new environment in the inventory (similar to referencing the tools)

woodbe commented:
I have updated based on the discussions for the camera as well as integrating the PAD-L2 suggestions.

- added a new environmental inventory table and defined a Normal Environment
- added reference to the Normal Environment in all the attacks as something that is needed
- updated the overview to point to the inventory document in the descriptions of the environment
- changed the smartphone camera rating to 130

Please review for the call next week.

woodbe commented:
Add to Normal Environment in the inventory to state that no shadows should be seen on the face use

woodbe committed:
Update L1-FACE_Toolbox_Inventory.adoc

Updated environment to include no harsh shadows
